src/models/Factor.js

```diff
@@ -165,14 +166,39 @@ function (Okta, Q, factorUtil, Util, Errors, BaseLoginModel) {
           return status === 'ACTIVE';
         }
       },
-      additionalEnrollment : {
-        deps: ['policy'],
-        fn: function (policy) {
-          if (!policy || !policy.enrollment) {
+      cardinality : {
+        deps: ['policy', 'profiles'],
+        fn: function (policy, profiles) {
+          if (profiles && profiles.length > 0) {
+            //there will only be one profile until we support multiple.
+            var profile = profiles[0];
+            var enrolled = profile._embedded.enrolledFactors.length;
```

Comment:
is it guaranteed on the API that we will have `_embedded.enrolledFactors`? Otherwise we can check that here and return false if that is missing

Reply:
That is a good question.. @AjayThomas-okta does the API guarantee we will have `_embedded.enrolledFactors`?

Reply:
only for factorTypes that support profiles will have it. 
others wont.
So, to begin with, only webauthn would and others like SMS, question wouldn't.
But if there's a profiles attribute for the factorType, then yes, it will have enrolledFactors always.

Reply:
Regarding your comment: "there will only be one profile until we support multiple."
For the record, we support multiple profiles in the framework in the backend.
But, webauthn factorType creates only one profile on enabling FF.
So, unless admin uses CRUD API to create more profiles, authn response would just return one profile since that's all that's to be fetched.

cc @antonioruberto-okta @jasleennuno-okta

Reply:
@nandagopalseshagiri-okta can you please confirm that for this round of changes we are always assuming that webauthn has only one profile?

Reply:
Yes, we are assuming that there is only one webauthn profile and it should be the default profile.